services/core-api/app/api/projects/namespace.py

```diff
@@ -85,6 +85,5 @@
 api.add_resource(ProjectDecisionPackageUploadedDocumentResource, '/<string:project_guid>/project-decision-package/<string:project_decision_package_guid>/documents/<string:mine_document_guid>')
 
 # Project Link
-api.add_resource(ProjectLinkListResource, '/<string:mine_guid>/project-link')
-api.add_resource(ProjectLinkListResource, '/<string:mine_guid>/project-link/<string:project_link_guid>')
-
+api.add_resource(ProjectLinkListResource, '/<string:project_guid>/project-link')
```

Comment:
I thought it was kind of wonky to go /projects/**mineGuid**/project-link, so I switched it out for projectGuid. I renamed the project_link_resource after picking the wrong file in my IDE too many times.